app/templates/posts.html

```diff
@@ -25,25 +25,34 @@ <h2> {{ users[post.userID] }} said:</h2>
 
     {% endif %}
     {% endif %}
-    <form action="" method="post">
-        {{ form.hidden_tag() }}
-        <p>
-            {% if form.title %}
-            {{ form.title.label }}<br>
-            {{ form.title() }}<br>
-            {% for error in form.title.errors %}
-            <span style="color: red;">[{{ error }}]</span>
-            {% endfor %}
-            {% endif %}
-
-            {{ form.post.label }}<br>
-            {{ form.post(cols=32, rows=4) }}<br>
-            {% for error in form.post.errors %}
-            <span style="color: red;">[{{ error }}]</span>
-            {% endfor %}
-        </p>
-        <p>{{ form.submit() }}</p>
-    </form>
+    <div class="post-heading">
+        <h1>Create A New Post</h1>
+    </div>
 
+    <div class="form-container">
+        <div class="post-form">
+            <h1 class="form-heading">Post An Announcement/Question!</h1>
+            <form action="" method="post">
+                {{ form.hidden_tag() }}
+                <div class="input-container">
+                    {% if form.title %}
+                    {{ form.title() }}
+                    {{ form.title.label }}
+                    {% for error in form.title.errors %}
+                    <span style="color: red;">[{{ error }}]</span>
+                    {% endfor %}
```

Comment:
See comment under `deleteClass.html`.
```suggestion
                    {% render_error(form.title) %}
```